README.md

```diff
@@ -29,23 +29,23 @@ Conceptlists in Concepticon:
 ![BIPA: 100%](https://img.shields.io/badge/BIPA-100%25-brightgreen.svg "BIPA: 100%")
 ![CLTS SoundClass: 100%](https://img.shields.io/badge/CLTS%20SoundClass-100%25-brightgreen.svg "CLTS SoundClass: 100%")
 
-- **Varieties:** 11
-- **Concepts:** 233
+- **Varieties:** 11 (linked to 9 different Glottocodes)
+- **Concepts:** 233 (linked to 0 different Concepticon concept sets)
```

Comment:
Something is off with the Concepticon mappings. Could you check the Lexibank scripts?

Reply:
Sorry, had unpushed changes in another branch. Fixed.